packages/dates/src/index.ts

```diff
@@ -10,4 +11,3 @@ export * from './is-tomorrow';
 export * from './parse-date-string';
 export * from './sanitise-date-string';
 export * from './unapply-time-zone-offset';
-export * from './map-deprecated-timezones';
```

Comment:
why'd this get pulled out?

Reply:
good question, it definitely should not be pulled out.